docs/rich_presence/Launch_Checklist.md

```diff
@@ -14,7 +14,7 @@ Ready to launch a Rich Presence integration for your game? Did you read our [Bes
 - Do your strings fit on their own lines without line wrapping?
   - Did you check on the smaller profile pop out?
 - Do they clearly communicate:
-  - What the player is currently doing?
+  - What is the player currently doing?
```

Comment:
this is not a question on its own - the intended reading is `Do they clearly communicate what the player is currently doing?`

Reply:
Okay, this is funny 😂  Thank you!